template.html

```diff
@@ -0,0 +1,238 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
```

Comment:
Noticed a lot of placeholder values, there should be something like `{name}` so it's easier to find and replace for the template generation